docker-entrypoint.sh

```diff
@@ -2,6 +2,9 @@
 set -Eeo pipefail
 # TODO swap to -Eeuo pipefail above (after handling all potentially-unset variables)
 
+# Define the path to the upgrade lock file using PGDATA if set, otherwise default
+UPGRADE_LOCK_FILE="${PGDATA:-/var/lib/postgresql}/upgrade_in_progress.lock"
```

Comment:
Oh.  I think the default path for `PGDATA` might need `/data` appended on the end there.  Will double check when I look at this properly in a few hours. :smile:

Reply:
Yeah. PGDATA should definitely be `/var/lib/postgresql/data` if nothing is passed.

Reply:
@justinclift will do it later, if im intrestet :P

Reply:
later is now xD

Reply:
Awesome, thanks. :smile: